app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -53,12 +55,14 @@ class MessageActivity : AppCompatActivity() {
     private val speed: Spinner by bindView(R.id.speed)
     private val mode: Spinner by bindView(R.id.mode)
     private val send: Button by bindView(R.id.send_button)
+    private val browse: Button by bindView(R.id.browse_button)
     private val previewButton: Button by bindView(R.id.preview_button)
     private val previewButtonDrawable: Button by bindView(R.id.preview_button_drawable)
     private val drawableRecyclerView: RecyclerView by bindView(R.id.recycler_view)
     private val sendByteLoader: ProgressBar by bindView(R.id.sendBytesLoader)
 
     private lateinit var drawableRecyclerAdapter: DrawableAdapter
+    private lateinit var listOfDrawables: ArrayList<DrawableInfo>
```

Comment:
We don't use lateinit var, or nullable unless it cannot be initialized right now

lateinit var or nullable - bad 
non-nullable val - good

Always use non-nullable val unless you absolutely cannot